tests/cpp/core/VisibilityHelperTests.cpp

```diff
@@ -211,3 +211,18 @@ BOOST_FIXTURE_TEST_CASE(testFullscreenWindowOverlapEverything, Fixture)
     BOOST_CHECK_EQUAL(helper.getVisibleArea(*focusWindow), QRectF());
     BOOST_CHECK_EQUAL(helper.getVisibleArea(*window), fullscreen);
 }
+
+BOOST_FIXTURE_TEST_CASE(testHiddenWindowNotObstructingOthers, Fixture)
+{
+    const QRectF& coord = window->getCoordinates();
+
+    ContentWindowPtr otherWindow = boost::make_shared<ContentWindow>(content);
+    group->addContentWindow(otherWindow);
+    otherWindow->setState(ContentWindow::WindowState::HIDDEN);
+    BOOST_CHECK_EQUAL(helper.getVisibleArea(*window), coord);
+
+    otherWindow->setState(ContentWindow::WindowState::NONE);
+    group->moveToFront(otherWindow);
```

Comment:
I expect moveToFront here has no effet. otherWindow is already in front otherwise the previous check does not make sense. Remove, it just obscures the intentions of the unit test.